scripts/entrypoint.sh

```
@@ -55,24 +55,51 @@ IFS="$OLD_IFS"
-done
-IFS="$OLD_IFS"
-
-chown -R "$APP_UID:$APP_GID" "$CRON_SPOOL_CACHE_DIR" >/dev/null 2>&1 || true
+	IFS="$OLD_IFS"
+	if [ -z "$WATCH_DIRS" ]; then
+		exit 0
+	fi
+	# close_write: editor saved a file. moved_to: atomic replace (crontab -e default).
+	# delete: user removed their crontab. Loop restarts inotifywait if it ever exits.
+	while true; do
+		inotifywait -qq -e close_write,moved_to,delete,create $WATCH_DIRS || sleep 5
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)